src/main/java/uk/gov/companieshouse/api/accounts/transformer/OffBalanceSheetArrangementsTransformer.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.api.accounts.transformer;
+
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.enumeration.AccountingNoteType;
+import uk.gov.companieshouse.api.accounts.model.entity.smallfull.notes.offbalancesheetarrangements.OffBalanceSheetArrangementsDataEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.smallfull.notes.offbalancesheetarrangements.OffBalanceSheetArrangementsEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.offbalancesheetarrangements.OffBalanceSheetArrangements;
+
+@Component
+public class OffBalanceSheetArrangementsTransformer implements NoteTransformer<OffBalanceSheetArrangements, OffBalanceSheetArrangementsEntity> {
```

Comment:
Can't find the usage of this class ?

Reply:
This is made available through the `NoteTransformerFactory`